app/models/member.rb

```diff
@@ -0,0 +1,8 @@
+class Member < ActiveRecord::Base
+  has_many :events
+  has_many :organizations
+  has_one :user
```

Comment:
@GalkaS I don't test the presence of associations specifically -- higher level tests will verify if this association exists -- i.e the app will blow up pretty quickly when doing a request spec for example.